event/handler_test.go

```diff
@@ -186,7 +194,7 @@ func TestImagePublishedHandler_Handle(t *testing.T) {
 			})
 
 			Convey("The image download variant is put to the image API with a state of completed", func() {
-				So(len(mockImageAPI.PutDownloadVariantCalls()), ShouldEqual, 1)
+				So(mockImageAPI.PutDownloadVariantCalls(), ShouldHaveLength, 1)
```

Comment:
Again, as above, Not an issue with this line (it's just the closest place I can put it) but after this Convey block, you could probably test that the image API's `Image` endpoints aren't called when everything is good. So something like…
```
Convey("The Image resource isn't modified directly", func() {
    So(mockImageAPI.GetImageCalls(), ShouldBeEmpty)
    So(mockImageAPI.PutImageCalls(), ShouldBeEmpty)
}
```